src/style.css

```diff
@@ -1 +1,48 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin:;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
```

Comment:
Don't use tag names for styling as per the checklist. Style the elements using '.classes'. If you need to apply the '.stars__star' styles to all children of '.stars', consider adding this class to each child element.